.github/workflows/release-drafter.yml

```
@@ -0,0 +1,24 @@
+name: Release Drafter
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - setup.py
+
+jobs:
+  create-release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - name: Get version number from setup.py 2
+        id: py-version
+        run: printf "version=%s\n" "$(grep '^[[:blank:]]*version' setup.py | cut -d'"' -f 2)" >> "$GITHUB_OUTPUT"
+      - uses: release-drafter/release-drafter@v6
+        with:
+          tag: "${{ steps.version.outputs.version }}"
```

```suggestion
          tag: "${{ steps.py-version.outputs.version }}"
```
Looking at the [docs](https://docs.github.com/en/actions/reference/workflows-and-actions/contexts#example-usage-of-the-jobs-context) you may have the reference the id `py-version` here